sass/components/home.scss

```diff
@@ -97,75 +96,115 @@
 @media screen and (max-width: 768px) {
 	.newsletter h3{
 		text-align: center;
+		margin-bottom: 40px;
 	}
 	.home-feature .container .col-md-4{
 		text-align:left;
 	}
-}
-@media screen and (max-width: 480px) {
+	.home-categories{
 
-.home-feature{
-	background: white;
-	min-height:200px;
-	.container{
-		padding-top: 0px;
-		.col-md-4{
-			padding-bottom:50px;
-			text-align: center;
-
-			h1{
-				color: rgb(93, 98, 102);
+		.home-box{
+			&:hover .home-box-content{
+				opacity: .7;
+				pointer-events: auto;
+				top:inherit;
 			}
-			p{
-				font-size:1.2em;
-				color:#acb1b4;
+			.home-box-content{
+				display: none;
+				position:absolute;
+				top:inherit;
+				width:100%;
+				max-height: 240px;
+				bottom: 0;
+
+				background-color: #333;
+				text-align: center;
+				opacity:.70;
+				pointer-events:none;
+
+				@include transition(0.2s);
+
+				.btn{
+					border:2px solid white;
+					color: #fff;
+				}
+
+				h2{
+					color: #fff;
+					font-weight: 600;
+				}
+				p{
+					color: #fff;
+					font-size:1.2em;
+		
+				}
 			}
 		}
 	}
 }
+@media screen and (max-width: 480px) {
 
-.home-categories{
-	.home-box{
-		&:hover .home-box-content{
-			opacity:1;
-			pointer-events: auto;
-			top:inherit;
-			bottom:0px;
+	.home-feature{
+		background: white;
+		min-height:200px;
+		.container{
+			padding-top: 0px;
+			.col-md-4{
+				padding-bottom:50px;
+				text-align: center;
+
+				h1{
```

Comment:
Spaces are needed between selectors `h1` and `{`. Like spaces between words in English!